test/unit/setup-wp-aliases.js

```diff
@@ -16,6 +16,7 @@ global.wp = {
 	'data',
 	'edit-post',
 	'viewport',
+	'plugins',
```

Comment:
This also needs to be added to the set in `bin/build-plugin-zip.sh` or it won't be included in the distributed plugin zip.

Reply:
Done